syntax="proto3";
package eos.console;

option go_package = "eos.rest.gateway.service";

//------------------------------------------------------------------------------
// Reply message sent to the client
//------------------------------------------------------------------------------
message ReplyProto {
  int64 retc      = 1; // return code of the command
  string std_err  = 2; // error message
  string std_out  = 3; // output message
}
